Automattic/wp-calypso
# Add Categories component to plugins browser

#### Changes proposed in this Pull Request

* Adds a new discover section (component named `Categories`)

#### Testing instructions

* http://calypso.localhost:3000/plugins
* Should see new discover section
* Should be able to click on discover dropdown
* Should be able to select categories

Related to https://github.com/Automattic/wp-calypso/issues/62264
